features/bevragen/persoon/nationaliteit/dev/rni-gba.feature

```diff
@@ -0,0 +1,35 @@
+#language: nl
+
+Functionaliteit: nationaliteit
+
+    Scenario: persoon heeft meerdere nationaliteiten aangeleverd door RNI deelnemer
+      Gegeven de persoon met burgerservicenummer '000000140' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) | rni-deelnemer (88.10) |
+      | 0263                  | 301                   | 19620107                        | 201                   |
+      En de persoon heeft nog een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) | rni-deelnemer (88.10) |
+      | 0052                  | 301                   | 19830326                        | 201                   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000140                       |
+      | fields              | nationaliteiten                 |
+      Dan heeft de response een persoon met een 'nationaliteit' met alleen de volgende gegevens
+      | naam                       | waarde                                   |
+      | nationaliteit.code         | 0263                                     |
+      | nationaliteit.omschrijving | Surinaamse                               |
+      | redenOpname.code           | 301                                      |
+      | redenOpname.omschrijving   | Vaststelling bezit vreemde nationaliteit |
+      | datumIngangGeldigheid      | 19620107                                 |
+      En heeft de persoon een 'nationaliteit' met alleen de volgende gegevens
+      | naam                       | waarde                                   |
+      | nationaliteit.code         | 0052                                     |
+      | nationaliteit.omschrijving | Belgische                                |
+      | redenOpname.code           | 301                                      |
+      | redenOpname.omschrijving   | Vaststelling bezit vreemde nationaliteit |
+      | datumIngangGeldigheid      | 19830326                                 |
+      En heeft de persoon een 'rni' met de volgende gegevens
```

Comment:
@CathyDingemanse @MelvLee @KayodeBakker dit is overgenomen uit #1544. Daarin heb ik voor het gemak de RNI 1 keer opgenomen, ook al komt het voor in 2 verschillende nationaliteiten. Je zou er ook voor kunnen pleiten dan 2 RNI items op te nemen (1 voor elke nationaliteit met een RNI-deelnemer). 
Theoretisch kan ook elke nationaliteit zijn aangeleverd door een andere deelnemer. Dan weet je niet meer welke van de 2 je moet opnemen. 

Wat is de eenvoudigste en beste oplossing?

Reply:
Zie scripts\upsert-landelijk-tabel-data-readme.md voor uitleg waarmee lokaal landelijke tabellen kan worden ge-update

Reply:
Graag even bespreken zometeen